pipeline/terraform/stack/locals.tf

```diff
@@ -74,7 +74,7 @@ locals {
   sierra_adapter_topic_arns = var.is_reindexing ? concat(var.adapters["sierra"].topics, [var.adapters["sierra"].reindex_topic]) : var.adapters["sierra"].topics
   miro_adapter_topic_arns   = var.is_reindexing ? concat(var.adapters["miro"].topics, [var.adapters["miro"].reindex_topic]) : var.adapters["miro"].topics
   mets_adapter_topic_arns   = var.is_reindexing ? concat(var.adapters["mets"].topics, [var.adapters["mets"].reindex_topic]) : var.adapters["mets"].topics
-  tei_adapter_topic_arns    = var.is_reindexing ? concat(var.adapters["tei"].topics, [var.adapters["tei"].reindex_topic]) : var.adapters["tei"].topics
+  tei_adapter_topic_arns    = []
```

Comment:
I wonder if doing this here is the right approach? This means you can never create a pipeline that transforms TEI works. Would it be better to have the topics commented out at the top level, so we can create a one-off pipeline to do TEI if we want to?

Either way, let's get a comment in somewhere explaining why we've removed this.